### PR TITLE
Split TS rules for infantry, structures and vehicles

### PR DESCRIPTION
**Pull request by reaperrr**
Less scrolling, more structured.

### CODE REVIEW

File: mods/ts/mod.yaml [hunk 1]
```diff
@@ -61,17 +61,26 @@ Packages:
...
+	./mods/ts/rules/civilian-vehicles.yaml
+	./mods/ts/rules/gdi-defenses.yaml
+	./mods/ts/rules/gdi-infantry.yaml
+	./mods/ts/rules/gdi-structures.yaml
+	./mods/ts/rules/gdi-vehicles.yaml
+	./mods/ts/rules/nod-defenses.yaml
+	./mods/ts/rules/nod-infantry.yaml
+	./mods/ts/rules/nod-structures.yaml
+	./mods/ts/rules/nod-vehicles.yaml
+	./mods/ts/rules/player-infantry.yaml
```
Comment (Phrohdoh) [inline mods/ts/mod.yaml]: How is this any different from `gdi-` and `nod-`? The name is also a bit misleading.

Comment (reaperrr) [inline mods/ts/mod.yaml]: ```shared-``` sounds good, that's pretty self-explanatory at first glance.

File: mods/ts/rules/civilian-structures.yaml [hunk 2]
```diff
@@ -1213,6 +1200,33 @@ GAOLDCC6:
 	RenderBuilding:
 		Palette: player
 
+GASAND:
+	Inherits: ^Wall
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 1000
+		Prerequisites: ~disabled
```
Comment (obrakmann) [inline mods/ts/rules/civilian-structures.yaml]: This had `~structures.gdi` before. Move it to `gdi-defenses.yaml` and fix the prerequisite?

Comment (pchote) [inline mods/ts/rules/civilian-structures.yaml]: This isn't supposed to be buildable by either faction.

**Comment by obrakmann** [comment]
Looks good to me, :+1:

**Comment by reaperrr** [comment]
Updated.
Renamed *-defenses to *-support and moved everything in the Defense queue into those files.

I didn't change the name of the queue itself for now since I suspect that to have a bigger chance of regressions, so I'd like to leave that for later.

**Comment by abcdefg30** [comment]
Needs a rebase.

**Comment by reaperrr** [comment]
Rebased.

**Comment by reaperrr** [comment]
Updated, removed pistol from civilian inf default and added it to civs 1 and 3 (civ2 has no shooting sequence).

**Comment by abcdefg30** [comment]
Shouldn't you also remove the sequences of `GAGREEN`?

**Comment by abcdefg30** [comment]
Looks otherwise ok to me and I didn't see anything ingame,
so :+1: / :white_check_mark: after above solved.

**Comment by abcdefg30** [comment]
Thanks. :)

**Pull request merged**